evalai/get_token.py

```diff
@@ -12,21 +13,5 @@ def get_token():
     """
     Get the EvalAI token.
     """
-    if not os.path.exists(AUTH_TOKEN_PATH):
-        echo(
-            style(
-                "\nThe authentication token json file doesn't exist at the required path. "
-                "Please download the file from the Profile section of the EvalAI webapp and "
-                "place it at ~/.evalai/token.json or use evalai -t <token> to add it.\n\n",
-                bold=True,
-                fg="red",
-            )
-        )
-    else:
-        with open(AUTH_TOKEN_PATH, "r") as fr:
-            try:
-                data = fr.read()
-                tokendata = json.loads(data)
-                echo("Current token is {}".format(tokendata["token"]))
-            except (OSError, IOError) as e:
-                echo(e)
+    token = get_user_auth_token()
+    echo(f"Current token is {token}")
```

Comment:
Can you please change this to `echo("Current token is {0}".format(token))` to maintain consistency in codebase?

Reply:
Done! :+1: